### PR TITLE
Update cn.h

### PR DESCRIPTION
**Pull request by lineteen**
Old Chinese language file is not complete?

---
This change is [Reviewable](https://reviewable.io/reviews/grayhatter/utox/263)

### CODE REVIEW

File: langs/cn.h [hunk 1]
```diff
@@ -5,81 +5,118 @@ msgid(LANG_ENGLISH_NAME)
...
+msgid(PROFILE_PW_NO_RECOVER)
+msgstr("忘记密码将无法找回。")
+
 msgid(ADDFRIENDS)
 msgstr("添加好友")
 
-//TODO: Same as in English? Localize or remove.
-//msgid(TOXID)
-//msgstr("Tox ID")
+msgid(TOXID)
```
Comment (GrayHatter) [inline langs/cn.h]: anything that uses the same text as the English version should be left as a comment.

Here and in a lot of others as well

File: langs/cn.h [hunk 2]
```diff
@@ -136,31 +244,44 @@ msgstr("语言")
...
-//msgid(IPV6)
-//msgstr("IPv6:")
+msgid(IPV6)
+msgstr("IPv6:")
 
-//TODO: Same as in English? Localize or remove.
-//msgid(UDP)
-//msgstr("UDP:")
+msgid(UDP)
+msgstr("UDP:")
```
Comment (GrayHatter) [inline langs/cn.h]: these too

**Comment by lineteen** [comment]
And :)
msgid(PROXY_EDIT_HINT_IP)
msgstr("IP")

**Comment by GrayHatter** [comment]
@lineteen  if you go here https://github.com/lineteen/uTox/blob/patch-1/langs/cn.h#L4 and click on the edit button, you can make these changes yourself. Then they'll automatically be changed here so that I can merge this.

**Comment by lineteen** [comment]
@GrayHatter Thanks! Edited.

**Comment by GrayHatter** [comment]
:lgtm:

I'll merge this once anyone else says it looks good!

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/grayhatter/utox/263#-:-KEEDvtv2140gvWTp9YA:2026120004)*

**Comment by stal888** [comment]
Rubber-stamping :lgtm:

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/grayhatter/utox/263#-:-KEbeFEl0noTzJ0dBP6o:498542805)*

**Comment by stal888** [comment]
*[langs/cn.h, line 29 \[r2\]](https://reviewable.io:443/reviews/grayhatter/utox/263#-KEbf8ZoPcJHKTLc-54R:-KEbf8ZoPcJHKTLc-54S:-545765328) ([raw file](https://github.com/grayhatter/utox/blob/6bcfd16df1aed12bad375f08c4d6da576430c173/langs/cn.h#L29)):*
Should be "Tox" here.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/grayhatter/utox/263)*

**Comment by GrayHatter** [comment]
@lineteen can you change the line that @stal888 commented?

**Comment by lineteen** [comment]
@GrayHatter @stal888 
Thanks for your help. I have changed that.

**Comment by stal888** [comment]
:lgtm:

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/grayhatter/utox/263#-:-KFAisV_352Phgvzjb4Y:1417088721)*

**Comment by GrayHatter** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/grayhatter/utox/263)*

**Comment by GrayHatter** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/grayhatter/utox/263#-:-KFFxOVhU25K0HN8r-JM:1417088721)*

**Pull request merged**